spec/datetimeformat.html

```diff
@@ -500,7 +503,10 @@
           The array that is the value of the "nu" property of any locale property of [[LocaleData]] must not include the values "native", "traditio", or "finance".
         </li>
         <li>
-          [[LocaleData]][locale] must have hourCycle property with String values for all locale values.
+          [[LocaleData]][locale].hc must be the array [*null*, *"h11"*, *"h12"*, *"h23"*, or *"h24"*] for all locale values.
```

Comment:
Yeah, I haven't seen such wording anywhere in 262 or 402. And reading it the second time, it is true that it is confusing. Does it have to be an array with 1) one of those values? or 2) a combination of those values? or 3) all of those values? I believe it is 1). @anba can you clarify? Maybe we should reword it.